package.json

```diff
@@ -46,12 +46,13 @@
   ],
   "main": "lib/Serverless.js",
   "bin": {
-    "serverless": "./bin/serverless",
-    "slss": "./bin/serverless",
-    "sls": "./bin/serverless"
+    "serverless": "./bin/serverless.js",
```

Comment:
Might not be a huge difference, but why don't we stick to the `./bin/serverless` file instead?

If we're using the `.js` file here, what is the `./bin/serverless` file used for?

Reply:
> Might not be a huge difference, but why don't we stick to the ./bin/serverless file instead?

File type discoverability is more complicated for many processors, e.g. ESLint, Prettier won't pick it and that means we're not guarded against errors as we should be.

I think it's a good reason to have extension on binary, and most projects I know switched to that (you may check: ESLint, Webpack, Prettier, npm etc.)

> If we're using the .js file here, what is the ./bin/serverless file used for?

It's kept to not accidentally break things for those which by chance refer to bin/serverless directly.

It's to be removed with next breaking version release. I explained that in a file: 

https://github.com/serverless/serverless/blob/bc128d5d269ce2227271491e211cade6f16f83dc/bin/serverless#L3-L5

Reply:
> File type discoverability is more complicated for many processors, e.g. ESLint, Prettier won't pick it and that means we're not guarded against errors as we should be.

Yep, I'm in the same boat. Was just wondering why we have such files but still keep the old one. However your comment regarding potential breaking changes makes sense (haven't seen the comment).